internal/compiler/widgets/common/spinbox-base.slint

```diff
@@ -35,27 +33,39 @@ export component SpinBoxBase {
 
     private property <length> scroll-delta: 2px;
 
-    forward-focus: i-text-input;
+    forward-focus: text-input;
+
+    changed value => {
+        root.set-text-to-value();
+    }
+
+    function set-text-to-value() {
+        text-input.text = root.value;
+    }
 
-    i-text-input := TextInput {
+    text-input := TextInput {
         vertical-alignment: center;
         horizontal-alignment: left;
         text: root.value;
         enabled: root.enabled;
         width: 100%;
         height: 100%;
 
+        accessible-role: AccessibleRole.text-input;
+        accessible-value: self.text;
+        accessible-label: "spinbox-textinput";
```

Comment:
I don't think that's a good use of accessible-label.

Now with this patch, I believe a screen reader would see two widgets with the same value, and one of them bing described as "spinbox-textinput" which will be hard for the user to understand.

Reply:
I talked with Simon about that topic. He showed me with the accessibility inspector on macOS that it is accessibility wise spread in different parts.

Reply:
So this was just a workaround anyway, with the id based search we can remove this label again.